src/lib/sql/sql_translator.cpp

```diff
@@ -441,8 +439,33 @@ std::shared_ptr<AbstractLQPNode> SQLTranslator::_translate_cross_product(const s
   return product;
 }
 
+std::shared_ptr<AbstractLQPNode> SQLTranslator::_apply_column_renames(const std::shared_ptr<AbstractLQPNode> node,
+                                                                      const hsql::TableRef& table) {
+  // Add a new projection node for table alias with column alias declarations
+  // e.g. select * from foo as bar(a,b)
+  if (table.alias && table.alias->columns) {
```

Comment:
Guard clauses are preferable, because they show that you perform an early-out and because they reduce indentation